site/src/components/gradients.md

```diff
@@ -17,12 +17,19 @@ There are two main types of gradients.
 
 ## Text color gradients
 
-Currently there is only one available gradient for text. Because gradients transitions take up the entire width of a particular element, it's recommended to highlight inline elements or a portion of a heading, not the entire heading itself.
+Because gradient transitions take up the entire width of a particular element. It's recommended to highlight inline elements, icons, or a portion of a heading, and not the entire heading itself.
```

Comment:
```suggestion
Because gradient transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, and not the entire heading itself.
```

Reply:
Please let commenters resolve their feedback.

In this case, combining two bits of grammar feedback led to something that also isn't grammatically correct - this should be one combined sentence, not split up like this.

Reply:
This new version makes it one sentence, so I agree with Ben's suggestion.

Reply:
Sorry on my old team the author resolved the feedback after it was addressed. I've updated this text.